### PR TITLE
Pm 2019 02 issues

### LINKED ISSUE #428: can't see full time when creating an event on small screen
bildschirmfoto 2018-07-13 um 10 57 54

### LINKED ISSUE #443: action icons are gone
screenshot from 2018-08-02 16-05-30

### LINKED ISSUE #593: delete button in dashboard document creation gone
screenshot from 2019-02-28 13-36-08

### PR DESCRIPTION
**Pull request by phillimorland**
fixes  #443
fixes #593 
fixes #428 
might have fixed following #599

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "feature-detect": "1.0.0",
     "file-loader": "3.0.1",
     "file-saver": "1.3.8",
+    "i": "^0.3.6",
```
Comment (fuzzylogic2000) [inline package.json]: what is this for?

File: package.json [hunk 2]
```diff
@@ -36,6 +37,7 @@
     "mapbox-gl-leaflet": "0.0.3",
     "mini-css-extract-plugin": "0.5.0",
     "node-sass": "4.11.0",
+    "npm": "^6.8.0",
```
Comment (fuzzylogic2000) [inline package.json]: do we normally have npm in the requirements? I thought not?

**Comment by phillimorland** [comment]
@fuzzylogic2000 we have $padding which is 1rem and $spacer which is 1em shall we leave them different? I have updated where I used it to $padding as should have been on first one anyway

**Comment by fuzzylogic2000** [comment]
> @fuzzylogic2000 we have $padding which is 1rem and $spacer which is 1em shall we leave them different? I have updated where I used it to $padding as should have been on first one anyway

@phillimorland Yes, we can leave it here like that. But for later, I thought, we could change it? I think, we added the r-spacer in mB, because we thought, there was no real use in a spacer in em, but were scared to replace it in the much-used mB. Here I am not that scared. I think, we would notice if it breaks anything (and could even chcek, where it's used and if that in turn is used in bet.in) and there are not too many people using it.

**Pull request merged**